internal/storage/minio_chunk_manager.go

```diff
@@ -102,16 +104,28 @@ func newMinioChunkManagerWithConfig(ctx context.Context, c *config) (*MinioChunk
 		Client:     minIOClient,
 		bucketName: c.bucketName,
 	}
-	log.Info("minio chunk manager init success.", zap.String("bucketname", c.bucketName), zap.String("root", c.rootPath))
+	mcm.rootPath = mcm.normalizeRootPath(c.rootPath)
+	log.Info("minio chunk manager init success.", zap.String("bucketname", c.bucketName), zap.String("root", mcm.RootPath()))
 	return mcm, nil
 }
 
+// normalizeRootPath
+func (mcm *MinioChunkManager) normalizeRootPath(rootPath string) string {
+	// no leading "/"
+	return strings.TrimLeft(rootPath, "/")
```

Comment:
Why not TrimLeft when param is initialized globally

Reply:
since this normalization is related to minio behavior only. It's very likely we gonna support `hdfs` or other storage services in the near future, so we want the normalization only happens in the chunkManager implementations.